src/d3a/models/market/__init__.py

```diff
@@ -124,6 +124,10 @@ def _create_fee_handler(self, grid_fee_type, grid_fees):
     def _is_constant_fees(self):
         return isinstance(self.fee_class, ConstantGridFees)
 
+    @property
+    def get_open_stock(self):
```

Comment:
Because bad names tend to stick around for a while, I would be precise and not invent nomenclature here, `get_open_bids_and_offers` or `list_open_bids_and_offers` are way more appropriate